overviewer_core/world.py

```diff
@@ -526,14 +524,31 @@ def __init__(self, regiondir, rel):
             'minecraft:quartz_stairs': (156, 0),
             'minecraft:activator_rail': (157, 0),
             'minecraft:dropper': (158, 0),
+			'minecraft:white_terracotta': (159, 0),
+            'minecraft:orange_terracotta': (159, 1),
+            'minecraft:magenta_terracotta': (159, 2),
+            'minecraft:light_blue_terracotta': (159, 3),
+            'minecraft:yellow_terracotta': (159, 4),
+            'minecraft:lime_terracotta': (159, 5),
+            'minecraft:pink_terracotta': (159, 6),
+            'minecraft:gray_terracotta': (159, 7),
+            'minecraft:silver_terracotta': (159, 8),
+            'minecraft:cyan_terracotta': (159, 9),
+            'minecraft:purple_terracotta': (159, 10),
+            'minecraft:blue_terracotta': (159, 11),
+            'minecraft:brown_terracotta': (159, 12),
+            'minecraft:green_terracotta': (159, 13),
+            'minecraft:red_terracotta': (159, 14),
+            'minecraft:black_terracotta': (159, 15),
```

Comment:
Can we continue handling these in the "for each color" loop toward the end of this function?